### PR TITLE
Add Getting Started text

### PR DESCRIPTION
**Pull request by emschwartz**

### CODE REVIEW

File: getting-started.html [hunk 1]
```diff
@@ -69,13 +69,33 @@
...
+          <p>First, create an account on the <a target="_blank"href="https://red.ilpdemo.org">Red Demo Wallet</a> and the <a target="_blank"href="https://blue.ilpdemo.org">Blue Demo Wallet</a> (one for sending and the other for receiving).</p>
 
-          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+          <div>
+          <img src="./assets/demo_register.png" />
```
Comment (bsteinlo) [inline getting-started.html]: @emschwartz I believe we can reference assets with just ` ` Does that work?

**Comment by emschwartz** [comment]
@bsteinlo You probably want to add more formatting to those images, or potentially replace them with higher quality ones, but you get the idea

File: getting-started.html (hunk 1 above)
Comment (emschwartz) [inline getting-started.html]: Changed

**Comment by bsteinlo** [comment]
LGTM

**Pull request merged**